src/tools/ceph-dencoder/types.h

```diff
@@ -1,3 +1,7 @@
+#pragma once
```

Comment:
ceph_dencoder.cc includes this header twice with different sets of macros, so the lack of include guards was intentional here

Reply:
@cbodley 
Could this lead to the fact that `ceph-dencoder` does not recognise any types any more???
Because if I run readble.sh it just fails ofr all types  returns like:
```
skipping unrecognized type uuid_d
skipping unrecognized type spg_t
skipping unrecognized type shard_id_t
skipping unrecognized type sobject_t
skipping unrecognized type watch_item_t
skipping unrecognized type watch_info_t
FAILED: no tests found to run!
```

Reply:
@tchaikov 
Removing the pragma restores the old behaviour, and lets `readable.sh` do its job again.

Reply:
thanks @cbodley and @wjwithagen . fixed and repushed.